messageBuilder.js

```diff
@@ -0,0 +1,65 @@
+const messages = {
+  prTitle: 'PR Title',
+  prBody: 'PR Description',
+  spaceBetweenHashtagAndDigit: 'PR contains a space between the hashtag and digit.',
+  noIssueReference: 'Issue Reference (`#<issue-number>`) missing.',
+  missingGithubKeyword: 'GitHub Keyword missing: Refer [here](https://help.github.com/articles/closing-issues-via-commit-messages/#keywords-for-closing-issues) for a list of accepted keywords.',
+};
+
+function isKeywordCheckerEnabled() {
+  return process.env.ENABLE_KEYWORD_CHECKER !== undefined &&
+    process.env.ENABLE_KEYWORD_CHECKER.toLowerCase() === 'true';
+}
+
+/*
+* Formats message as a GFMD level two unordered list item
+*/
+function getFormattedMessageLevelTwoUnordered(message) {
+  return `   * ${message}\n`;
+}
+
+/*
+* Formats message as a GFMD level two unordered list item
+*/
+function getFormattedMessageLevelOneOrdered(message) {
+  return `1. ${message}\n`;
+}
+
+function buildTitleFeedback(violations) {
+  let message = '';
+  if (violations === undefined) return message;
```

Comment:
I think a `winston.warn` that violations is undefined might be warranted here because it'll make it hard to debug in the future when `violations` is forgotten to be passed as an input.

Although an error would be better, I don't think you would want to bring down a bot crashing because it could build a proper message 😂